### PR TITLE
Add point display

### PR DESCRIPTION
**Pull request by kfischer-okarin**
Screen Shot 2019-04-20 at 12 36 40 PM

Solves #36

### CODE REVIEW

File: app/assets/javascripts/app/components/metric.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  def before_create
+    @label = option_replace :label
+    @value = option_replace :value
+    @color = option_replace :color
+  end
+
+  def cascade
+    add_child :metric_value, Value, content: @value
+    metric_value.add_state @color, true if @color
+    # add_child :label, Label, content: @label
```
Comment (fkdkent) [inline app/assets/javascripts/app/components/metric.rb]: This line can be removed? 👀

File: app/assets/javascripts/app/pages/connections/index.js.rb [hunk 2]
```diff
@@ -80,13 +83,17 @@ def add_connection(connection_json)
...
-      @points[result.team] += 1
+      update_points(result)
     }
   end
 
   def content
     add_child :network_container, Panel, title: 'Status'
     @network = network_container.add_content :network, Network, nodes: nodes, edges: edges, options: NETWORK_OPTIONS
+    points = network_container.add_to_header :points, Points
+    @point_display = TEAMS.map { |t|
```
Comment (fkdkent) [inline app/assets/javascripts/app/pages/connections/index.js.rb]: This can be written as `TEAMS.to_h { |t| [t, points.add_child(...)] }`.

Comment (kfischer-okarin) [reply app/assets/javascripts/app/pages/connections/index.js.rb]: it seems Opal does not support this version of to_h yet

File: app/assets/javascripts/app/components/metric.rb (hunk 1 above)
Comment (kfischer-okarin) [reply app/assets/javascripts/app/components/metric.rb]: Fixed in c8033e1

File: app/assets/javascripts/app/pages/connections/index.js.rb (hunk 2 above)
Comment (fkdkent) [reply app/assets/javascripts/app/pages/connections/index.js.rb]: So we have a chance to send a pull request 🚀

**Pull request merged**